lib/matplotlib/offsetbox.py

```diff
@@ -1554,6 +1580,9 @@ def disconnect(self):
         for disconnector in self._disconnectors:
             disconnector()
 
+        if self._hover:
+            self.canvas.set_cursor(Cursors.POINTER)
```

Comment:
```suggestion
        if self._hover:
            self._hover = False
            self.canvas.set_cursor(Cursors.POINTER)
```

Reply:
Is currently redundant because `self._hover` is never accessed after the callbacks have been disconnected, but can do in case that changes in the future.